repo-projects/Directory.Build.props

```diff
@@ -227,4 +227,24 @@
     <ProjectReference Include="$(TasksDir)Microsoft.DotNet.UnifiedBuild.MSBuildSdkResolver\Microsoft.DotNet.UnifiedBuild.MSBuildSdkResolver.csproj" ReferenceOutputAssembly="false" SkipGetTargetFrameworkProperties="true" Condition="'$(DotNetBuildSourceOnly)' != 'true'" />
   </ItemGroup>
 
+  <!-- Runtime repository references. These repositories are excluded from the build when we're not building runtime components. -->
+  <ItemGroup>
+    <RuntimeRepositoryReference Include="aspire" />
+    <RuntimeRepositoryReference Include="aspnetcore" />
+    <RuntimeRepositoryReference Include="cecil" />
+    <RuntimeRepositoryReference Include="command-line-api" />
+    <RuntimeRepositoryReference Include="deployment-tools" />
+    <RuntimeRepositoryReference Include="diagnostics" />
+    <RuntimeRepositoryReference Include="efcore" />
+    <RuntimeRepositoryReference Include="emsdk" />
+    <RuntimeRepositoryReference Include="runtime" />
+    <RuntimeRepositoryReference Include="source-build-externals" />
```

Comment:
That repo doesn't exist anymore. Are you maybe on an old HEAD?

Reply:
> I assume both the SBRP and the SBE content will now always get built in 2xx, 3xx, 4xx?

That's correct.